pages/linux/pacman-sync.md

```diff
@@ -0,0 +1,36 @@
+# pacman sync
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Install a new package:
+
+`pacman --sync {{package_name}}`
+
+- Synchronize and update all packages (add `--downloadonly` for download but do not update anything):
```

Comment:
Wouldn't it make more sense to _replace_ `--sysupgrade` with `--downloadonly`, rather than add `--downloadonly`?

Reply:
Without `--sysupgrade`, `pacman` doesn't download the packages. The `--refresh` only updates the local repositories (similar to `apt update`)

Reply:
Then, how about...
```suggestion
- Synchronize and update all packages (add `--downloadonly` to download only):
```